tests/src/test/java/tachyon/client/LocalBlockInStreamIntegrationTest.java

```diff
@@ -94,7 +107,6 @@ public void readTest1() throws IOException, TachyonException {
         Assert.assertEquals(cnt, k);
         Assert.assertTrue(BufferUtils.equalIncreasingByteArray(k, ret));
         is.close();
-        Assert.assertTrue(sFileSystem.getStatus(uri).getInMemoryPercentage() == 100);
```

Comment:
why is the in-memory percentage not checked anymore? (here and and other test cases in this class)